Update viewer tutorial 3D mode docs (#159)
napari/docs: docs/tutorials/fundamentals/viewer.md

> #### 2D/3D button  

Author: ```suggestion
#### Toggle ndisplay (2D/3D) button  
```

> The second button from the left is the 2D/3D button which toggles between `2D` and `3D` renderings of the data. 

Reviewer: I would add that its tooltip says "Toogle ndisplay", it's not immediately obvious. However now I'm thinking we should probably do this for all buttons - maybe in a follow-up PR?

Author: Should we change the heading to `Toggle ndisplay (2d/3d) button`?

Author: Done

viewer.add_labels(labeled, name='blob ID')
```

then, by clicking the 2D/3D button, you can switch to viewing the 3D rendering. Note that the icon will change to
the following, to indicate 3D mode: 

![image: 3D_button](../assets/tutorials/3D_button.png)

> Note that this mode can be entered programmatically using:

Reviewer: To avoid duplication of "note":
```suggestion
This mode can be entered programmatically using:
```

viewer.camera.angles = (3, 38, 53)
nbscreenshot(viewer, alt_text="A rotated 3D view")
```

> Note that if you want to drag the rendering itself, instead of rotating the view, you have tohold down the 

Author: ```suggestion
Note that if you want to drag the canvas/rendering itself, instead of rotating the view, you have to hold down the 
```

viewer.camera.angles = (3, 38, 53)
nbscreenshot(viewer, alt_text="A rotated 3D view")
```

Note that if you want to drag the canvas/rendering itself, instead of rotating the view, you have to hold down the 
Shift key while dragging with the mouse. Finally, while in 3D mode you can change the perspective of the 

> 3D view by right-clicking on the 2D/3D mode button, which will bring up the perspective slider.

Author: ```suggestion
3D view by holding Shift, pressing the right mouse button (on macOS holding Control) and
dragging the mouse or by right-clicking (on macOS holding Control and clicking) on the 2D/3D mode
button, which will bring up the perspective slider.
```

viewer.camera.angles = (3, 38, 53)
nbscreenshot(viewer, alt_text="A rotated 3D view")
```

Note that if you want to drag the canvas/rendering itself, instead of rotating the view, you have to hold down the 
Shift key while dragging with the mouse. Finally, while in 3D mode you can change the perspective of the 
3D view by holding Shift, pressing the right mouse button (on macOS holding Control) and
dragging the mouse or by right-clicking (on macOS holding Control and clicking) on the 2D/3D mode

> button, which will bring up the perspective slider.

Author: ```suggestion
button, which will bring up the perspective slider. The camera perspective can also be altered programmatically:
```python
viewer.camera.perspective = 45
```
```
